tests/stage/__snapshots__/test_diff.ambr

```diff
@@ -0,0 +1,197 @@
+# serializer version: 1
+# name: test_print_diff_to_console_no_bundlemap
+  '''
+  Your stage is up-to-date with your local deploy root
+  
+  '''
+# ---
+# name: test_print_diff_to_console_no_bundlemap.1
+  '''
+  Your stage is up-to-date with your local deploy root
+  
+  '''
+# ---
+# name: test_print_diff_to_console_no_bundlemap.2
+  '''
+  Deleted paths to be removed from your stage:
+    deleted:  deleted
+    deleted:  dir/deleted
+    deleted:  dir/only_on_stage
+    deleted:  only_on_stage
+  
+  '''
+# ---
+# name: test_print_diff_to_console_no_bundlemap.3
+  '''
+  Local changes to be deployed:
+    modified: different
+    modified: dir/different
+    modified: dir/modified
+    modified: modified
+  Deleted paths to be removed from your stage:
```

Comment:
Suggestion for consistency with `git status`: leave a blank line between groups.